### PR TITLE
Add "code suggestion" recommendation to review docs

### PR DESCRIPTION
**Pull request by craig-landry**
@andrablaj and I were talking about using this feature and she rightly pointed out that documenting it is the best approach to team-wide adoption (rather than Slack messages here and there).

For now this seems to be the best place for this recommendation, but ideally we have a set of more detailed pages on code reviews. We currently link out to other recommendations (cool, since there's no need to duplicate all that), but it means not really having a good landing page for all this. For the time being I'm adding this here, but if we expand our code review guidance, this section should move to whatever comes in the future.

### CODE REVIEW

File: content/en/contribute/code/workflow.md [hunk 1]
```diff
@@ -28,13 +28,31 @@ A good workflow would be to work locally, pushing to a remote branch as you make
 
 The author and reviewer should use this [guide to code reviewing](https://google.github.io/eng-practices/review/developer/).
 
+#### Suggestions
+
+When doing a code review aim to be extremely clear. This helps things move quickly and avoids lost time in misunderstandings. One especially useful GitHub feature for doing this is suggesting a change. Consider the following example code:
```
Comment (andrablaj) [inline content/en/contribute/code/workflow.md:33]: Should we link the GitHub [related documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request)?

**Review by andrablaj — APPROVED** [review]

File: content/en/contribute/code/workflow.md (hunk 1 above)
Comment (craig-landry) [reply content/en/contribute/code/workflow.md:33]: > Should we link the GitHub [related documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request)?

Yes! Although, I couldn't find a GitHub source for guidance on this feature specifically. I found a couple pages when it went into beta in 2018 and more general stuff (like the link you mentioned), but I could not find any page from GitHub on this feature. Have you seen anything better @andrablaj?

Comment (andrablaj) [reply content/en/contribute/code/workflow.md:33]: [Step 6](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) of the link I shared is about the _suggestion_ feature. I didn't find anything more detailed though.

Comment (andrablaj) [reply content/en/contribute/code/workflow.md:33]: The PR can be merged without applying this suggestion, as your documentation is more detailed than the GitHub guide.

**Pull request merged**